fighthealthinsurance/settings.py

```
@@ -334,6 +335,15 @@ class Base(Configuration):
         "django_prometheus.middleware.PrometheusAfterMiddleware",
     ]
 
+    # Django's default tag for messages.ERROR is "error", but Bootstrap 5 (the
+    # version base.html loads) has no `.alert-error` class — only
+    # `.alert-danger`. Without this remap the flash-message block in base.html
+    # renders every error as unstyled body text, which defeats the point of
+    # showing it. Every messages.* call in this codebase is messages.error().
+    MESSAGE_TAGS = {
+        messages_constants.ERROR: "danger",
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)